pulp_puppet_plugins/pulp_puppet/plugins/importers/importer.py

```diff
@@ -77,12 +78,8 @@ def import_units(self, source_repo, dest_repo, import_conduit, config, units=Non
         return copier.copy_units(import_conduit, units)
 
     def upload_unit(self, repo, type_id, unit_key, metadata, file_path, conduit, config):
-        try:
-            report = upload.handle_uploaded_unit(repo, type_id, unit_key, metadata, file_path,
-                                                 conduit)
-        except Exception, e:
-            _logger.exception(e)
-            report = {'success_flag': False, 'summary': e.message, 'details': {}}
```

Comment:
+1 to removing the Exception altogether. The machinery that catches it in tasks.py is going to do all the same things this does. With it present the same traceback will be logged twice though.